sdk/storage/storage-file-share/src/generated/src/storageClientContext.ts

```diff
@@ -11,7 +11,7 @@
 import * as coreHttp from "@azure/core-http";
 
 const packageName = "azure-storage-file-share";
-const packageVersion = "12.0.0";
+const packageVersion = "12.0.0-preview.7";
```

Comment:
12.0.0 is the latest released version. we should bump the version number.

Reply:
Same for blob. Probably should pass it via build:autorest command line option.

Reply:
Yeah. Changed the script command for autorest in package.json and re-generated.